src/test/java/com/github/lombrozo/testnames/rules/ml/RulePresentSimpleMlTest.java

```diff
@@ -89,6 +90,34 @@ void checksCorrectNames(final String name) {
         );
     }
 
+    /**
+     * Test case for present tense on plural in test methods.
+     * @param name Test method name
+     * @todo #248:25min Enable passesOnPlural test when plural speech detection will
+     *  be implemented in {@link RulePresentSimpleMl}. Let's test that the following
+     *  pattern: $pluralNoun_$pluralVerb... passes.
+     */
+    @Disabled
+    @CsvSource(
+        {
+            "theyBuildModel",
+            "theyPassTest",
+            "robotsDoGood",
+            "documentsServePeople"
+        }
+    )
+    @ParameterizedTest
+    void passesOnPlural(final String name) {
```

Comment:
@h1alexbel Since we already have `checksCorrectNames` test that do exactly the same I would indicate somehow that this `passesOnPlural` test serves different purpose:

1. You can change the name of the test `passesOnPlural` -> `passesOnPluralWithSubjectAtTheBeginning` (or something like that)
2. You can change the assertion message `"Name '%s' has to be correct"` -> `"Here we expect that the test name with a subject at the beginning will be correct, but it wasn't '%s'"` 
3. Maybe something else

Or you can just add these test cases to the `checksCorrectNames` test.

What do you think?

Reply:
@h1alexbel Also I would add a bit more examples:
```java
    "humansCraftSolutions",
    "machinesLearnPatterns",
    "usersTrustSystem",
    "algorithmsPredictOutcome",
    "clientsReceiveNotifications",
    "serversHandleRequest",
    "employeesSubmitReport",
    "devicesSyncData",
    "botsRespondInstantly",
    "studentsCompleteAssignments",
    "applicationsRunSmoothly",
    "tasksGenerateResults",
    "administratorsManageAccess",
    "dataDrivesDecisions",
    "systemsProcessInput",
    // Two-word examples
    "dogsBark",
    "catsPurr",
    "birdsSing",
    "fishSwim",
    "treesGrow",
    "riversFlow",
    "carsRace",
    "childrenPlay",
    "studentsStudy"
```

Reply:
@volodya-lombrozo I would like to segregate new test case for plural and already exists `checksCorrectNames` since we can't disable plural inputs for now (though we can comment out them).